frontend/src/components/sidebar/active.jsx

```diff
@@ -32,7 +32,8 @@ class ActiveSidebar extends React.Component {
                                         currentUserId={currentUserId}
                                         history={history} 
                                         activeTask={this.props.activeTask}
-                                        receiveActiveTaskId={this.props.receiveActiveTaskId}/>
+                                        receiveActiveTaskId={this.props.receiveActiveTaskId}
+                                        currentPosition={this.props.currentPosition}/>
```

Comment:
For future stuff, try using destructing for variables that are being pulled out of the same object, like props.

Reply:
hell yea thanks my dude:)
will do illr refactor on my next push